src/mapper/postTransformations.ts

```diff
@@ -0,0 +1,131 @@
+import cytoscape, { ElementDefinition, Position } from 'cytoscape';
+import { NodeType } from '../models/nodeType';
+import { getSymbol } from '../symbol-api/getSymbol';
+import { NodeSymbol } from '../symbol-api/types/NodeSymbol';
+import { SymbolRotation } from '../symbol-api/types/SymbolRotation';
+import { arrayEquals } from '../utils/arrayEquals';
+import { mergeElementsByKey } from './mergeElements';
+import { postProcessSvgTag } from './transformations';
+
+export type Transformation = {
+	transformNew: (element: ElementDefinition) => ElementDefinition[];
+	transformUpdate: (element: ElementDefinition, cy: cytoscape.Core) => void;
+};
+
+export const createSvgTransformation = (iconNode2Connectors: { [iconNode: string]: ElementDefinition[] }): Transformation => {
+	const transform = (element: ElementDefinition) => {
+		const rotation = parseInt(element.data.rotation) as SymbolRotation;
+		const symbol = getSymbol(element.data.symbolId, { rotation: rotation });
```

Comment:
```suggestion
		const { height, width, connectors } = getSymbol(element.data.symbolId, { rotation: rotation });
```

Reply:
I am using symbol below so I would prefer to keep it. Also this function got a bit big so it is nice to have the `symbol.height`-context instead of just `height`. Do you agree?